mailgreen/services/subscription_service.py

```
@@ -0,0 +1,139 @@
+    import requests
+
+    sub = db.query(Subscription).filter_by(id=sub_id).first()
+    if not sub:
+        raise ValueError("Subscription not found")
+
+    link = sub.unsubscribe_link
+    host = urlparse(link).netloc
+    path = urlparse(link).path
+
+    if link.lower().startswith("mailto:"):
+        from mailgreen.services.mail_service import send_mail_via_gmail_api
```

Pylint (3.3.7)

[error] No name 'send_mail_via_gmail_api' in module 'mailgreen.services.mail_service'

(E0611)